### PR TITLE
subscribe loki channel dynamically and increase ebpf memory limit for CD workload

### PR DESCRIPTION
**Pull request by memodi**
- Loki channels keeps changing in redhat-operators catalog, this will allow us to subscribe to latest channel.
- ebpf-agent on infra nodes crashes when CD + ingress workload is run, this PR also increases the ebpf memory for CD workload.

### CODE REVIEW

**Comment by memodi** [comment]
/cc @Amoghrd 
tested here: https://jenkins-csb-openshift-qe-mastern.dno.corp.redhat.com/job/scale-ci/job/memodi-e2e-benchmarking-multibranch-pipeline/job/loki-channel/3/console (it's currently running but it past the loki operaror subscribe stage)

```
12-13 11:37:25.904  + echo '====> Creating openshift-operators-redhat Namespace and OperatorGroup'
12-13 11:37:25.904  ====> Creating openshift-operators-redhat Namespace and OperatorGroup
12-13 11:37:25.904  + oc apply -f /home/jenkins/ws/workspace/ultibranch-pipeline_loki-channel/ocp-qe-perfscale-ci/scripts/loki/loki-operatorgroup.yaml
12-13 11:37:26.157  namespace/openshift-operators-redhat created
12-13 11:37:26.158  operatorgroup.operators.coreos.com/openshift-operators-redhat created
12-13 11:37:26.158  + echo '====> Creating netobserv-downstream-testing CatalogSource (if applicable) and Loki Operator Subscription'
12-13 11:37:26.158  ====> Creating netobserv-downstream-testing CatalogSource (if applicable) and Loki Operator Subscription
12-13 11:37:26.158  + export LOKI_CHANNEL=
12-13 11:37:26.158  + LOKI_CHANNEL=
12-13 11:37:26.158  + [[ Released == \U\n\r\e\l\e\a\s\e\d ]]
12-13 11:37:26.158  ++ get_loki_channel redhat-operators
12-13 11:37:26.158  ++ catalog_label=redhat-operators
12-13 11:37:26.158  +++ oc get packagemanifests -l catalog=redhat-operators -n openshift-marketplace -o 'jsonpath={.items[?(@.metadata.name=="loki-operator")].status.channels[*].name}'
12-13 11:37:27.079  ++ channels=stable-6.1
12-13 11:37:27.079  ++ read -r -a channels_arr
12-13 11:37:27.079  ++ len=1
12-13 11:37:27.079  ++ echo stable-6.1
12-13 11:37:27.079  + LOKI_CHANNEL=stable-6.1
12-13 11:37:27.079  + subscription=/home/jenkins/ws/workspace/ultibranch-pipeline_loki-channel/ocp-qe-perfscale-ci/scripts/loki/loki-released-subscription.yaml
12-13 11:37:27.079  + '[' -z stable-6.1 ']'
12-13 11:37:27.079  + echo '====> Using Loki chanel stable-6.1 to subscribe'
12-13 11:37:27.079  ====> Using Loki chanel stable-6.1 to subscribe
12-13 11:37:27.079  + envsubst
12-13 11:37:27.079  + oc apply -f -
12-13 11:37:27.333  subscription.operators.coreos.com/loki-operator created
12-13 11:37:27.333  + echo '====> Generate S3_BUCKET_NAME'
12-13 11:37:27.333  ====> Generate S3_BUCKET_NAME
12-13 11:37:27.333  ++ tr -dc a-z0-9
12-13 11:37:27.333  ++ fold -w 6
12-13 11:37:27.334  ++ head -n 1
12-13 11:37:27.334  + RAND_SUFFIX=zctu9s
12-13 11:37:27.334  + [[ cluster-density-v2 == \N\o\n\e ]]
12-13 11:37:27.334  + [[ -z cluster-density-v2 ]]
12-13 11:37:27.334  + export S3_BUCKET_NAME=netobserv-ocpqe-memodi-cluster-density-v2-zctu9s
12-13 11:37:27.334  + S3_BUCKET_NAME=netobserv-ocpqe-memodi-cluster-density-v2-zctu9s
12-13 11:37:27.334  ++ oc get infrastructure cluster -o 'jsonpath={.status.infrastructureName}'
12-13 11:37:27.334  + CLUSTER_NAME=memodi-perf-95ksb
12-13 11:37:27.334  + [[ memodi-perf-95ksb =~ preserve ]]
12-13 11:37:27.334  + echo '====> S3_BUCKET_NAME is netobserv-ocpqe-memodi-cluster-density-v2-zctu9s'
12-13 11:37:27.334  ====> S3_BUCKET_NAME is netobserv-ocpqe-memodi-cluster-density-v2-zctu9s
12-13 11:37:27.334  + echo '====> Creating S3 secret for Loki'
12-13 11:37:27.334  ====> Creating S3 secret for Loki
```

File: scripts/netobserv.sh [hunk 1]
```diff
@@ -106,16 +116,22 @@ deploy_lokistack() {
...
   echo "====> Creating netobserv-downstream-testing CatalogSource (if applicable) and Loki Operator Subscription"
-  if [[ $LOKI_OPERATOR == "Released" ]]; then
-    oc apply -f $SCRIPTS_DIR/loki/loki-released-subscription.yaml
-  elif [[ $LOKI_OPERATOR == "Unreleased" ]]; then
+  export LOKI_CHANNEL=''
+  if [[ $LOKI_OPERATOR == "Unreleased" ]]; then
     deploy_downstream_catalogsource
-    oc apply -f $SCRIPTS_DIR/loki/loki-unreleased-subscription.yaml
+    LOKI_CHANNEL=$(get_loki_channel qe-app-registry)
+    subscription=$SCRIPTS_DIR/loki/loki-released-subscription.yaml
```
Comment (Amoghrd) [inline scripts/netobserv.sh:123]: this should use unreleased subscription right?

Comment (memodi) [reply scripts/netobserv.sh]: good catch. I fixed and simplify code in my latest commit.

**Comment by memodi** [comment]
here's the success run where ebpf memory was increased: https://jenkins-csb-openshift-qe-mastern.dno.corp.redhat.com/job/scale-ci/job/memodi-e2e-benchmarking-multibranch-pipeline/job/loki-channel/8/consoleFull

/unhold

**Pull request merged**